ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md

```diff
@@ -863,6 +863,9 @@ tenants:
           < IPv4 dhcp server IP >:
             source_interface: < interface-name >
             source_vrf: < VRF to originate DHCP relay packets to DHCP server. If not set, uses current VRF >
+        
+        # Manually define the VLAN used on the MLAG pair for the iBGP session
+        custom_mlag_ibgp_peering_vrfs: <1-4096>
```

Comment:
I suggest renaming to `mlag_ibgp_peering_vlan: <1-4096> -> default : <base_vlan> + <vrf_vni>`